### PR TITLE
feat(Template): Set up Standalone with Submodule Connection + Models

### PR DESCRIPTION
**Pull request by Elaina-Lee**
- **Please check if the PR fulfills these requirements**

* [X] The commit message follows our guidelines

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Set up standalone for template
- **What is the current behavior?** (You can also link to an open issue here)

- **What is the new behavior (if this is a feature change)?**

- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

- **Please Include Screenshots or Videos of the intended change**:

### CODE REVIEW

File: libs/designer/src/lib/ui/templates/connections/displayConnections.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Button } from '@fluentui/react-components';
+import type { Template } from '@microsoft/logic-apps-shared';
+
+export interface DisplayConnectionsProps {
+  connections: Record<string, Template.Connection>;
+}
+
+export const DisplayConnections = ({ connections }: DisplayConnectionsProps) => {
+  return (
+    <div>
```
Comment (ccastrotrejo) [inline libs/designer/src/lib/ui/templates/connections/displayConnections.tsx:10]: Nit: maybe use empty tags?

File: libs/designer/src/lib/ui/templates/connections/displayConnections.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+              <Button
+                appearance="outline"
+                onClick={() => {
+                  console.log('TODO: show connection for this');
+                }}
+              >
+                CHOOSE
+              </Button>
+            </b>
+            <div>
```
Comment (ccastrotrejo) [inline libs/designer/src/lib/ui/templates/connections/displayConnections.tsx:26]: NIT: do you need to a div inside an empty div? I mean maybe move the inner div outside?

File: libs/designer/src/lib/ui/templates/connections/displayConnections.tsx (hunk 1 above)
Comment (Elaina-Lee) [reply libs/designer/src/lib/ui/templates/connections/displayConnections.tsx:10]: This component is just a placeholder for displaying connections and will have to be changed again, but I can take those NIT in account to keep it clean!

File: libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  const templateConnections = useSelector((state: RootState) => state.template.connections);
+
+  return (
+    <>
+      <div>AVAILABLE MANIFEST NAMES</div>
+      {availableManifestsNames?.map((manifestName: string) => (
+        <div key={manifestName}>{manifestName}</div>
+      ))}
+      <div>CURRENTLY SELECTED TEMPLATE</div>
+      {templateConnections ? <DisplayConnections connections={templateConnections} /> : <>no connections to be made</>}
```
Comment (ccastrotrejo) [inline libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx:16]: NIT: maybe use intl in the `no connections to be made`?

Comment (Elaina-Lee) [reply libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx:16]: We will definitely remember to bring in intl when we want to display texts to the users! This is a placeholder that will be deleted

File: libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import { useSelector } from 'react-redux';
+import { DisplayConnections } from './connections/displayConnections';
+
+export const TemplatesDesigner = () => {
+  const availableManifestsNames = useSelector((state: RootState) => state.manifest.availableManifestNames);
+  const templateConnections = useSelector((state: RootState) => state.template.connections);
+
+  return (
+    <>
+      <div>AVAILABLE MANIFEST NAMES</div>
```
[the remark this answers is not in the document]
Comment (Elaina-Lee) [reply libs/designer/src/lib/ui/templates/TemplatesDesigner.tsx:11]: We will definitely remember to bring in intl when we want to display texts to the users! This is a placeholder that will be deleted

**Review by ccastrotrejo — APPROVED** [review]

**Pull request merged**